llvm/test/Transforms/LoopVectorize/AArch64/force-inst-cost-invalid.ll

```diff
@@ -0,0 +1,24 @@
+; REQUIRES: asserts
+; RUN: opt < %s -passes=loop-vectorize -force-target-instruction-cost=1 -debug-only=loop-vectorize -S -disable-output 2>&1 | FileCheck %s
+target triple = "aarch64-linux-gnu"
+
+define i32 @invalid_legacy_cost(i64 %N) #0 {
+; CHECK: LV: Checking a loop in 'invalid_legacy_cost
+; CHECK: LV: Found an estimated cost of Invalid for VF vscale x 2 For instruction: %0 = alloca i8, i64 0, align 16
```

Comment:
This is checking the legacy cost model print I think, you want to check the VPlan printing or just check the generated IR

Reply:
Thanks, I'm now just checking the generated IR after moving the test.